app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,55 @@
 <resources>
+
+    <!-- Strings related to the name of each activity page -->
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
     <string name="app_name">Gather Words</string>
     <string name="session_title">Session</string>
     <string name="entry_title">Lexical Entry</string>
+    <string name="app_settings_page_title">Settings</string>
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
+
+
+
+    <!-- Strings related to the toolbar/action bar -->
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
+    <string name="app_action_bar_settings">Settings</string>
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
+
+
+
+    <!-- Strings for TextView objects -->
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
+    <string name="app_main_page_title">Hello, Gather Words!</string>
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
+
+
+
+    <!-- Strings for the language selection list-type preference setting -->
+    <!-- =============================================================== -->
+    <!-- =============================================================== -->
+    <string name="pref_language_select_key">list_pref_1anguage</string>
+    <string name="pref_language_select_title">Language Selection.</string>
```

Comment:
`<string name="pref_language_select_key">list_pref_1anguage</string>
 <string name="pref_language_select_title">Language Selection.</string>`

Delete the periods from the end of these strings